dspace/config/registries/openaire-cerif-types.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0"?>
+
+<dspace-dc-types>
+
+	<dspace-header>
+		<title>OpenAIRE Cris Types</title>
+	</dspace-header>
+
+	<dc-schema>
+		<name>oairecerif</name>
+		<namespace>https://www.openaire.eu/cerif-profile/1.1/</namespace>
+	</dc-schema>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>identifier</element>
+        <qualifier>url</qualifier>
+        <scope_note></scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>person</element>
+        <qualifier>gender</qualifier>
+        <scope_note></scope_note>
+    </dc-type>
+    
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>acronym</element>
+        <scope_note></scope_note>
+    </dc-type>
+    
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>internalid</element>
+        <scope_note></scope_note>
```

Comment:
Needs a scope note? Are we sure this is required and cannot be achieved via a dc.identifier field?